modules/@ergonode/authentication/src/components/Layout/Login.vue

```diff
@@ -97,26 +112,32 @@ export default {
 
         &__body {
             position: relative;
-            display: flex;
+            display: grid;
             flex: 1;
             justify-content: center;
-            align-items: center;
+            grid-auto-flow: column;
+            grid-template-columns: 1fr 1fr;
+            grid-column-gap: 16px;
+            width: 100%;
+            height: 100%;
```

Comment:
height 100% is not recommended to set with flex-direction: column - already setting flex: 1 is expanding for full height